### PR TITLE
New Tests added

### PR DESCRIPTION
**Pull request by programmer4web**
fixes https://github.com/Neovici/cosmoz-image-viewer/issues/25

code coverage from 35% to
Statements   : 91.96% ( 103/112 )
Branches     : 87.5% ( 42/48 )
Functions    : 96.15% ( 25/26 )
Lines        : 91.89% ( 102/111 )

### CODE REVIEW

**Comment by programmer4web** [comment]
The tests include openFullcsreen test, I have created an fix for that https://github.com/Neovici/cosmoz-image-viewer/pull/27

**Comment by programmer4web** [comment]
@plequang This is ready for Review.

**Comment by programmer4web** [comment]
@nomego This is ready for review.

**Comment by programmer4web** [comment]
@plequang I think this is ready for merging if you agree. :)

File: cosmoz-image-viewer.js [hunk 1]
```diff
@@ -232,13 +232,13 @@
...
 		_createImageOverlay() {
-			let dialog = document.createElement('cosmoz-image-viewer-overlay');
+			const dialog = document.createElement('cosmoz-image-viewer-overlay');
 			dialog.id = 'cosmoz-image-viewer-overlay';
 			dialog.noCancelOnOutsideClick = true;
 			dialog.loop = this.loop;
 			dialog.showDetach = this.showDetach;
 			document.body.appendChild(dialog);
-			imageOverlay = dialog;
+			imageOverlay = this.imageOverlay = dialog;
```
Comment (plequang) [inline cosmoz-image-viewer.js]: Didn't you removed `this.imageOverlay` in another PR ?
There is no `imageOverlay` property in this element

Comment (programmer4web) [reply cosmoz-image-viewer.js]: @plequang I did but it is needed for tests and I omitted that aspect at that moment.

Comment (programmer4web) [reply cosmoz-image-viewer.js]: @plequang I have added:
get imageOverlay() {
       return imageOverlay;
},

**Pull request merged**